### ISSUE TITLE
Not working in Jupyterlab

### ISSUE THREAD

**Issue by davidbrochart** [maartenbreddels/ipyvolume#102]
Hi Maarten,

I was trying to use ipyvolume in Jupyterlab, but I get the message "Failed to display Jupyter Widget of type VBox". Here is my setup:

```
git clone https://github.com/maartenbreddels/ipyvolume.git
mkdir ipyvolume_env
cd ipyvolume_env
pipenv install jupyterlab
pipenv shell
pipenv install -e ../ipyvolume
jupyter nbextension install --py --symlink --sys-prefix ipyvolume
jupyter nbextension enable --py --sys-prefix ipyvolume
jupyter lab ../ipyvolume/notebooks/example.ipynb
```

**Comment by maartenbreddels** [comment]
Hi David,

that means that jupyter-widgets is not installed into jupyter lab (the js part of the library), this nbextensions command is only for the notebook (although all of this will be done automagically in the future).
Try running this:
```
$ jupyter labextension install @jupyter-widgets/jupyterlab-manager
```
Then see if you get say a slider working
```python
import ipywidgets
ipywidgets.FloatSlider()
```
If that works, try enabling ipyvolume:
```
$ jupyter labextension install ipyvolume
```
Let me know if this works out!

**Comment by davidbrochart** [comment]
Yes it works fine. Thanks a lot!

**Comment by maartenbreddels** [comment]
:+1:

**Comment by davidbrochart** [comment]
So it looks like `jupyter labextension install ipyvolume` fetches https://registry.npmjs.org/ipyvolume. Is there a way to have everything locally when developing an IPython widget?

**Comment by maartenbreddels** [comment]
Yes, I hope the next version will include it in the source distribution, we plan to discuss this next week at a workshop actually. Also, the last argument can be a directory:
https://jupyterlab-tutorial.readthedocs.io/en/latest/developer/extension_dev.html

**Comment by AHochstadt** [comment]
Hello, I've encountered basically all the errors possible on this ticket and others and have tried different versions and solutions to no avail. 

I'm installing the extension from a clean conda environment that I've made specially for JupyterLab. I'm on Linux with Google Chromium as the default Jupyter browser.

I am able to get the ipywidgets FloatSlider successfully after `jupyter labextension install @jupyter-widgets/jupyterlab-manager`:
```
import ipywidgets
ipywidgets.FloatSlider()
```

I'm also able to install the threejs extension with `jupyter labextension install jupyter-threejs` successfully (or at least without breaking JupyterLab).

But if I run `jupyter labextension install ipyvolume`, my JupyterLab invariably breaks: when I run `jupyter lab`, I just get a blank white screen with 

```
lab.js:107 Uncaught TypeError: Cannot read property 'default' of undefined
    at f.registerPluginModule (lab.js:107)
    at index.out.js:1552
    at Array.forEach (<anonymous>)
    at a (index.out.js:1552)
```
in the javascript console.

I would tell you all my version numbers, but I've tried too many to really enumerate them. Let me know if you want additional info about my latest attempt or in general and I will be happy to provide.  

Appreciate it. I'm really excited to use this amazing package.
